lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -289,7 +297,19 @@ def self.available_options
                                   env_name: "APPCENTER_DISTRIBUTE_RELEASE_NOTES_LINK",
                                description: "Additional release notes link",
                                   optional: true,
-                                      type: String)
+                                      type: String),
+
+          FastlaneCore::ConfigItem.new(key: :build_number,
+                                       env_name: "APPCENTER_DISTRIBUTE_BUILD_NUMBER",
+                                       description: "Android apk build number",
```

Comment:
I would suggest change the description here to: `The build number. Used (and required) for uploading Android ProGuard mapping file.`